docs/sql-programming-guide.md

```diff
@@ -1362,6 +1362,13 @@ options.
  - Dataset and DataFrame API `explode` has been deprecated, alternatively, use `functions.explode()` with `select` or `flatMap`
  - Dataset and DataFrame API `registerTempTable` has been deprecated and replaced by `createOrReplaceTempView`
 
+ - Changes to `CREATE TABLE ... LOCATION` behavior.
+    - From Spark 2.0, `CREATE TABLE ... LOCATION` is equivalent to `CREATE EXTERNAL TABLE ... LOCATION`
+      in order to prevent accidental dropping the existing data in the user-provided locations.
```

Comment:
Also add two more sentences here. `That means, a Hive table created in Spark SQL with the user-specified location is a Hive external table. Users are not allowed to specify the location for Hive managed tables. `